src/inference/src/ie_network_reader.cpp

```diff
@@ -520,6 +504,23 @@ CNNNetwork details::ReadNetwork(const std::string& modelPath,
     std::string FEs;
     for (const auto& fe_name : manager.get_available_front_ends())
         FEs += fe_name + " ";
```

Comment:
I suppose the creation of error message should also be moved below

Reply:
Maybe here can prompt some warning info about available frontend cannot handle, need ir_v7_loader to handle it.

Reply:
@ilyachur why do you ignore this comment?